server/monitor-types/zabbix-trigger.js

```diff
@@ -0,0 +1,64 @@
+const { MonitorType } = require("./monitor-type");
+const { UP, DOWN } = require("../../src/util");
+const axios = require("axios");
+
+class ZabbixTriggerMonitorType extends MonitorType {
+
+    name = "zabbix-trigger";
+
+    /**
+     * @inheritdoc
+     */
+    async check(monitor, heartbeat, _server) {
+
+        // Make Request
+        const options = {
+            method: "post",
+            url: monitor.zabbixInstanceUrl,
+            data: {
+                "jsonrpc": "2.0",
+                "method": "trigger.get",
+                "params": {
+                    "triggerids": monitor.zabbixTriggerId,
+                    "output": "extend",
+                    "selectFunctions": "extend",
```

Comment:
Seems like `selectFunctions` is only adding extra infos about `functions`, which we don't use.
Am I reading this wrong or can we remove this?